bigquery_etl/cli/query.py

```diff
@@ -493,6 +493,14 @@ def info(ctx, name, sql_dir, project_id, cost, last_updated):
         click.echo("")
 
 
+def _parse_parameter(parameter: str, param_date: str) -> str:
+    # TODO: Parse more complex parameters such as macro_ds_add
+    if "{{ds}}" not in parameter:
```

Comment:
This should ideally use the same parsing logic as airflow does, as an import or something